Unable to build from git in Arch Linux

Trying to build current master, and getting errors on building 0rt. The end of the error is here, full log attached.

[385/976] Building CXX object CMakeFiles/onnxruntime_session.dir/home/ngoonee/.cache/aurutils/sync/obs-backgroundremoval-git/src/obs-backgroundremoval/build/Ort-prefix/src/Ort/onnxruntime/core/session/inference_session.cc.o
ninja: build stopped: subcommand failed.
Traceback (most recent call last):
  File "/home/ngoonee/.cache/aurutils/sync/obs-backgroundremoval-git/src/obs-backgroundremoval/build/Ort-prefix/src/Ort/tools/ci_build/build.py", line 2812, in <module>
    sys.exit(main())
  File "/home/ngoonee/.cache/aurutils/sync/obs-backgroundremoval-git/src/obs-backgroundremoval/build/Ort-prefix/src/Ort/tools/ci_build/build.py", line 2727, in main
    build_targets(args, cmake_path, build_dir, configs, num_parallel_jobs, args.target)
  File "/home/ngoonee/.cache/aurutils/sync/obs-backgroundremoval-git/src/obs-backgroundremoval/build/Ort-prefix/src/Ort/tools/ci_build/build.py", line 1349, in build_targets
    run_subprocess(cmd_args, env=env)
  File "/home/ngoonee/.cache/aurutils/sync/obs-backgroundremoval-git/src/obs-backgroundremoval/build/Ort-prefix/src/Ort/tools/ci_build/build.py", line 740, in run_subprocess
    return run(*args, cwd=cwd, capture_stdout=capture_stdout, shell=shell, env=my_env)
  File "/home/ngoonee/.cache/aurutils/sync/obs-backgroundremoval-git/src/obs-backgroundremoval/build/Ort-prefix/src/Ort/tools/python/util/run.py", line 49, in run
    completed_process = subprocess.run(
  File "/usr/lib/python3.10/subprocess.py", line 526, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/usr/bin/cmake', '--build', '/home/ngoonee/.cache/aurutils/sync/obs-backgroundremoval-git/src/obs-backgroundremoval/build/Ort-prefix/src/Ort-build/Release', '--config', 'Release', '--', '-j16']' returned non-zero exit status 1.
make[2]: *** [CMakeFiles/Ort.dir/build.make:89: Ort-prefix/src/Ort-stamp/Ort-build] Error 1
make[1]: *** [CMakeFiles/Makefile2:114: CMakeFiles/Ort.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
[errors.txt](https://github.com/royshil/obs-backgroundremoval/files/10952878/errors.txt)

--- a/scripts/PKGBUILD
+++ b/scripts/PKGBUILD
@@ -1,44 +1,22 @@
 pkgname=obs-backgroundremoval
-pkgver=0.4.0
-_ONNXRUNTIME_FILE=onnxruntime-linux-x64-1.7.0.tgz
-_onnxdir=onnxruntime-linux-x64-1.7.0
-_bgdir=$pkgname-${pkgver//_/-}
+pkgver=0.5.17
 pkgrel=1
+pkgdesc="Background removal plugin for OBS studio"
 arch=(x86_64)
 url='https://github.com/royshil/obs-backgroundremoval'
-pkgdesc="Background removal plugin for OBS studio (precomopiled onnxruntime)"
-license=(MIT custom)
-depends=(obs-studio opencv)
+license=(GPL2)
+depends=(obs-studio)
 makedepends=(cmake)
-source=("${_bgdir}.tar.gz::https://github.com/royshil/$pkgname/archive/refs/tags/v${pkgver//_/-}.tar.gz"
-        "https://github.com/microsoft/onnxruntime/releases/download/v1.7.0/$_ONNXRUNTIME_FILE")
-sha256sums=('e88bbc902154b5efd46d8fc1dee52fcf079c8ff2ae6593d9f716017fdd1ef601'
-            '0345f45f222208344406d79a6db3280ed2ccc884dc1e064ce6e6951ed4c70606')
-
-prepare() {
-  # build from archive, not git. Version set during build()
-  sed -i "s/^version_from_git/#&/" $_bgdir/CMakeLists.txt
-}
+source=("https://github.com/royshil/$pkgname/archive/refs/tags/v$pkgver.tar.gz")
+sha256sums=('7e44c352f2b1d082f7904fea4773caadf52cc666e8c64ed954a69a57c31c855b')
 
 build() {
-  cd "$_bgdir"
-  # set rpath to avoid installing onnxruntime globally
-  cmake -B build -DVERSION="${pkgver//_*/.0}" -DobsIncludePath=/usr/include/obs/ \
-           -DCMAKE_INSTALL_PREFIX:PATH=/usr \
-           -DOnnxruntime_INCLUDE_DIRS="$srcdir/$_onnxdir/include/" -DOnnxruntime_LIBRARIES="$srcdir/$_onnxdir/lib/libonnxruntime.so" \
-           -DOnnxruntime_INCLUDE_DIR="$srcdir/$_onnxdir/include/" \
-           -DCMAKE_INSTALL_RPATH="/usr/lib/obs-backgroundremoval"
-  cd build
-  make
+  cd "$pkgname-$pkgver"
+  cmake -B build -DLINUX_PORTABLE=OFF
+  cmake --build build
 }
 
 package() {
-  make -C "$_bgdir/build" DESTDIR="$pkgdir" install
-  install -Dt "$pkgdir/usr/share/licenses/obs-backgroundremoval" "$_bgdir/LICENSE"
-
-  # install onnxruntime
-  cd "$_onnxdir"
-  install -Dt "$pkgdir/usr/share/licenses/obs-backgroundremoval/onnxruntime" \
-          LICENSE  Privacy.md  README.md  ThirdPartyNotices.txt
-  install -Dt "$pkgdir/usr/lib/obs-backgroundremoval" lib/*
+  cd "$pkgname-$pkgver"
+  cmake --install build --prefic "$pkgdir/usr"
 }

--- a/scripts/PKGBUILD
+++ b/scripts/PKGBUILD
@@ -18,5 +18,5 @@ build() {
 
 package() {
   cd "$pkgname-$pkgver"
-  cmake --install build --prefic "$pkgdir/usr"
+  cmake --install build --prefix "$pkgdir/usr"
 }

--- a/scripts/PKGBUILD
+++ b/scripts/PKGBUILD
@@ -5,14 +5,14 @@ pkgdesc="Background removal plugin for OBS studio"
 arch=(x86_64)
 url='https://github.com/royshil/obs-backgroundremoval'
 license=(GPL2)
-depends=(obs-studio)
+depends=(obs-studio opencv)
 makedepends=(cmake)
 source=("https://github.com/royshil/$pkgname/archive/refs/tags/v$pkgver.tar.gz")
 sha256sums=('7e44c352f2b1d082f7904fea4773caadf52cc666e8c64ed954a69a57c31c855b')
 
 build() {
   cd "$pkgname-$pkgver"
-  cmake -B build -DLINUX_PORTABLE=OFF
+  cmake -B build -DLINUX_PORTABLE=OFF -DUSE_SYSTEM_OPENCV=ON
   cmake --build build
 }
 

